test/test-barrier.c

```diff
@@ -104,3 +104,46 @@ TEST_IMPL(barrier_3) {
 
   return 0;
 }
+
+static void serial_worker(void* data) {
+  uv_barrier_t* barrier = (uv_barrier_t*) data;
+  if (uv_barrier_wait(barrier) > 0) {
+    uv_barrier_destroy(barrier);
```

Comment:
Might as well ASSERT this succeeds, here and elsewhere.

Reply:
My mistake. I posted this before noticing that returned void. I assumed it was just a pass-through to `pthread_barrier_destroy` with the same signature.